src/design-tokens/tier-2-usage/typography-usage.css

```diff
@@ -20,6 +20,14 @@
   @mixin eds-typography-preset-005-bold $important;
 }
 
+@define-mixin eds-theme-typography-body-sm $important {
+  @mixin eds-typography-preset-006-bold $important;
+}
+
+@define-mixin eds-theme-typography-body-xs $important {
+  @mixin eds-typography-preset-008-bold $important;
+}
```

Comment:
aren't there existing `eds-theme-typography-body-text-sm-bold` and `eds-theme-typography-body-text-xs-bold` presets (I think at the bottom of the file)? Could we move those up & make sure they're pointing to the right presets

Reply:
ahh yeah weird, I think that never got updated 🤔  since `body-text-xs` is pointing to 008 now